### ISSUE TITLE
Joystick control is not precise (standalone, not Retroarch)

### ISSUE THREAD

**Issue by mdeguzis** [Javanaise/mrboom-libretro#15]
I noticed the game only seems to respond to joystick control on the Xbox 360 controller, and often seems to have imprecise movement, or the character will try to walk up on his own.

**Comment by frranck** [comment]
I may be doing something stupid here https://github.com/libretro/mrboom-libretro/blob/master/sdl2.c#L216

**Comment by frranck** [comment]
Do you have an example of SDL2 game working fine with your controllers?

**Comment by mdeguzis** [comment]
I'll double check my packages I build for ones that depend on SDL2, that's as much as I can dig up.

**Comment by frranck** [comment]
@mdeguzis Hi, could you please try again with current master ?

**Comment by mdeguzis** [comment]
sorry, been busy, I should have free time tongith.

**Comment by frranck** [comment]
Any update?

**Comment by mdeguzis** [comment]
Much better it seems, very few occasions where the direction "doubles back" (going up, quick switch to down). I'd say this is ok for me. I cannot get sound to go beyond muffled burts, but i'll raise another ticket for that. I built off the master tree today.

**Comment by mdeguzis** [comment]
One thing to note though, is I can't used the dpad on the Xbox 360 controller. Joystick is fine enough as it is for now.
